TECHNICAL-STEERING-COMMITTEE.md

```diff
@@ -83,10 +93,14 @@ The TSC may invite non-members to participate in a non-voting capacity. These in
  
 The moderator is responsible for summarizing the discussion of each agenda item and publishing it on a publicly accessible platform (e.g., the forum). If appropriate, the moderator will also update the relevant issue, pull request or forum post.
  
-**Decision Making**
+## <a name="decision-making"></a>Decision Making
 
 For internal project decisions, Committers shall operate under Lazy Consensus. The TSC shall establish appropriate guidelines for implementing Lazy Consensus (e.g., expected notification and review time periods).
  
 The TSC follows a Consensus Seeking decision-making model. When an agenda item has appeared to reach a consensus, the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
  
 If an agenda item cannot reach a consensus, a TSC member can call for either a closing vote or a vote to table the issue to the next meeting. The call for a vote must be approved by a majority of the TSC or else the discussion will continue. If all members of the TSC are not present during the meeting for contentious issues, the final vote should happen asynchronously (e.g., via email). Simple majority wins.
+
+## <a name="attribution"></a>Attribution
+
+This a derivative work of the [Node.js Project Governance](https://github.com/nodejs/nodejs.org/blob/HEAD/locale/en/about/governance.md). This work is licensed under the [MIT](https://opensource.org/licenses/MIT) license.
```

Comment:
I would prefer this actually point to a specific version rather than to HEAD so we can go back to the actual version we forked from. It's the current version now but would be great to be able to easily see any changes since the fork.